src/identity/types.ts

```diff
@@ -59,3 +59,5 @@ export type GetAvatarReturnType = string | null;
  * Note: exported as public Type
  */
 export type GetNameReturnType = string | null;
+
+export type SchemaId = `0x${string}`;
```

Comment:
What's the difference with `EASSchemaUid`? if they are the same let's just use `EASSchemaUid`